README.md

```diff
@@ -37,22 +46,135 @@ api("com.github.spotify:openfeature-kotlin-sdk:[ANY_BRANCH]-SNAPSHOT")
 
 This will get a build from the head of the mentioned branch. 
 
-### Resolving a flag
+### Maven
+
+Installation via Maven Central is currently WIP
+
+## 🌟 Features
+
+- support for various backend [providers](https://openfeature.dev/docs/reference/concepts/provider)
+- easy integration and extension via [hooks](https://openfeature.dev/docs/reference/concepts/hooks)
+- bool, string, numeric, and object flag types
+- [context-aware](https://openfeature.dev/docs/reference/concepts/evaluation-context) evaluation
+
+## 🚀 Usage
+
+```kotlin
+    // configure a provider and get client
+    OpenFeatureAPI.setProvider(customProvider)
+    val client = OpenFeatureAPI.getClient()
+
+    // get a bool flag value
+    client.getBooleanValue("boolFlag", default = false)
+    
+    // get a bool flag value async
+    coroutineScope.launch {
+        WithContext(Dispatchers.IO) {
+            client.awaitProviderReady()
+        }
+        client.getBooleanValue("boolFlag", default = false)
+    }
+```
+
+### Events
+
+Events allow you to react to state changes in the provider or underlying flag management system, such as flag definition changes, provider readiness, or error conditions.
+Initialization events (`PROVIDER_READY` on success, `PROVIDER_ERROR` on failure) are dispatched for every provider.
+Some providers support additional events, such as `PROVIDER_CONFIGURATION_CHANGED`.
+Please refer to the documentation of the provider you're using to see what events are supported.
+
 ```kotlin
-import dev.openfeature.sdk.*
+    OpenFeatureAPI.eventsObserver()
+        .observe<OpenFeatureEvents.ProviderReady>()
+        .collect {
+            // do something once the provider is ready
+        }
+```
+
+### Providers:
+
+To develop a provider, you need to create a new project and include the OpenFeature SDK as a dependency.
+This can be a new repository or included in [the existing contrib repository](https://github.com/open-feature/java-sdk-contrib) available under the OpenFeature organization.
```

Comment:
This is linking to the java sdk contrib now. We'll need to wait for the kotlin contrib location to be ready before adding it to the documentation

Reply:
There's another place in the readme that links to Kotlin repo (which isn't populated yet) - let's keep in mind to update these upon release.